src/mlir/generator.cc

```diff
@@ -130,6 +130,14 @@ namespace mlir::verona
           return std::move(err);
         // Associate function with module (late mangling)
         func->setAttr("class", TypeAttr::get(type));
+        // Add qualifiers as attributes
```

Comment:
None of the tests has a non-empty list of qualifiers.  What is a qualifier in this context?

Reply:
some do have the "static" qualifier, but it's hard to see in the sea of red and green... :)

they're not used yet by the compiler, but we'll use them later to check if a call is static or dynamic. 

I could have checked the type of the first argument (`DescriptorType` or not) but attributes are more stable than values (casts, etc), and we must support them for every operation, so doesn't cost to have them, since the info is in the AST anyway.